lib/inferno/dsl/fhir_evaluator/config/default.yml

```diff
@@ -0,0 +1,80 @@
+# Common configuration.
```

Comment:
From discussion - this file should be as empty as possible for the initial impl

Reply:
Keep configurations only actively used.
Removed all Rule configurations for now as we don't run any rules for now.